Mathlib/CategoryTheory/Limits/Shapes/Biproducts.lean

```diff
@@ -547,6 +555,28 @@ theorem biproduct.isoCoproduct_hom {f : J → C} [HasBiproduct f] :
     (biproduct.isoCoproduct f).hom = biproduct.desc (Sigma.ι f) :=
   biproduct.hom_ext' _ _ fun j => by simp [← Iso.eq_comp_inv]
 
+/-- If a category has finite biproducts, its `colim` and `lim` functor on finite diagrams
+are isomorophic. -/
+@[simps!]
+def HasFiniteBiproducts.colimIsoLim [HasFiniteBiproducts C] [Finite J] :
```

Comment:
This proof is the same as the one below, it looks like that one should make this one quite a bit easier? You'll need that `HasBiproductsOfShape J C` is implied by `Finite J` and `HasFiniteBiproducts C`, but that's fine: `hasBiproductsOfShape_finite`